R/z_animint.R

```diff
@@ -357,7 +357,7 @@ animint2dir <- function(plot.list, out.dir = NULL,
       }
       geom_num <- geom_num + 1
       layer_name <- getLayerName(L, geom_num, p.name)
-      gl <- Geom$export_animint(L, df, meta, layer_name,
+      gl <- L$geom$export_animint(L, df, meta, layer_name,
```

Comment:
is this correct? why do we have to refer to L twice? seems like we should fix that.

Reply:
or if export_animint method can't be fixed maybe keep the original saveLayer function.

Reply:
Actually we're doing some operations/pre-processing in the export_animint() function by using L.

Reply:
looking at the original OO ggplot code it seems like the usual/consistent syntax is actually what you had originally, Geom$export_animint so I guess we should use that. sorry for the confusion.

Reply:
So, Should I go ahead and revert the last change and use `Geom$export_animint()` instead?